types/account.d.ts

```diff
@@ -236,19 +247,20 @@ export type Capture = {
 };
 
 export type UpdateCaptureAnswersPayload = {
-  customerId: string;
+  customer: Customer;
 } & Capture;
 
 // TODO: Convert these all to generic non-cleeng calls
-type Login = (args: LoginArgs) => Promise<{ auth: AuthData; user: Customer }>;
-type Register = CleengRequest<RegisterPayload, AuthData>;
-type GetPublisherConsents = CleengRequest<GetPublisherConsentsPayload, GetPublisherConsentsResponse>;
-type GetCustomerConsents = CleengAuthRequest<GetCustomerConsentsPayload, GetCustomerConsentsResponse>;
+// type Login = CleengRequest<LoginPayload, AuthData>;
+type Login = (args: AuthArgs) => Promise<{ auth: AuthData; user: Customer }>;
+type Register = (args: AuthArgs) => Promise<{ auth: AuthData; user: Customer }>;
+type GetPublisherConsents = ServiceRequest<Config, GetPublisherConsentsResponse>;
+type GetCustomerConsents = ServiceRequest<GetCustomerConsentsPayload, GetCustomerConsentsResponse>;
```

Comment:
ServiceRequest is more concise, so that's fine. Just be careful to decouple the `Payload` and `Response` objects from the args that we use in the services, because they define 1-1 the Cleeng api contract at the moment and we don't want to break that. I think it would also be nice to change the services to just throw errors instead of returning an array of errors in `ApiResponse` which we then just rethrow anyhow.